storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/BaseSubscriptionTest.java

```diff
@@ -0,0 +1,40 @@
+package com.pushtorefresh.storio.content_resolver.impl;
+
+import android.support.annotation.NonNull;
+
+import java.util.Queue;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import rx.Subscription;
+
+public abstract class BaseSubscriptionTest extends BaseTest {
+
+    public abstract class BaseSubscribeStub<T> {
+        private final Queue<T> expected;
+        private CountDownLatch lock;
+
+        public BaseSubscribeStub(@NonNull Queue<T> expected) {
+            this.expected = expected;
+            lock = new CountDownLatch(expected.size());
+        }
+
+        public boolean syncWait() {
+            try {
+                lock.await(2000, TimeUnit.MILLISECONDS);
```

Comment:
Hm, hardcoded time is pretty dangerous, if emulator will slow down -> test can fail :(

May be we can loop with waits and set max loop time, if changes will arrive faster -> break the loop, also it will make tests faster